my-project/public/css/Shared/coreStyle.css

```diff
@@ -0,0 +1,16 @@
+/**
+ * VI-38 CoreStyles
+ *
+ * @category   Styles
+ * @package    Virtua_CoreStyles
+ * @copyright  Copyright (c) Virtua
+ * @author     Mateusz Soboń <m.sobon@wearevirtua.com>
+ */
+
```

Comment:
@mSobonVirtua add specific class to the page and than to the css here so only images on the category page gallery have those dimentions, even better create a separate file with styles for category, example: categoryStyle.css

Reply:
But I want to share this style, because @dkruczek may want use it. This classes should be global.

Reply:
ok, than lets leave it